src/index.html

```diff
@@ -7,6 +7,344 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#logo" class="header__logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a class="nav__link is-active" href="#apple">Apple
+            </a>
+          </li>
+          <li>
+            <a class="nav__link" href="#samsung">Samsung
+            </a>
+          </li>
+          <li>
+            <a class="nav__link" href="#smartphones">Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="#laptops"
+              data-qa="nav-hover"
+            >Laptops & computers
+            </a>
+          </li>
+          <li>
+            <a class="nav__link" href="#gadgets">Gadgets
+            </a>
+          </li>
+          <li>
+            <a class="nav__link" href="#tables">Tablets
+            </a>
+          </li>
+          <li>
+            <a class="nav__link" href="#photo">Photo
+            </a>
+          </li>
+          <li>
+            <a class="nav__link" href="#video">Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="product">
```

Comment:
I guess it means exactly `html` tag
https://github.com/mate-academy/layout_catalog/pull/2448#discussion_r1051331576

but you can set width to the cards container (now it's `.product`)
because it looks not good now (on big screen):
![image](https://user-images.githubusercontent.com/86923089/209212417-23d195e3-f187-45d7-a873-9a2a2aaf1a58.png)